include/eigensolve_quda.h

```diff
@@ -136,34 +133,32 @@ namespace quda
        @param[in] kSpace The Krylov space vectors
        @param[in] evals The eigenvalue array
     */
-    void cleanUpEigensolver(std::vector<ColorSpinorField *> &kSpace, std::vector<Complex> &evals);
+    void cleanUpEigensolver(std::vector<ColorSpinorField> &kSpace, std::vector<Complex> &evals);
 
     /**
-       @brief Applies the specified matVec operation:
-       M, Mdag, MMdag, MdagM
+       @brief Promoted the specified matVec operation:
+       M, Mdag, MMdag, MdagM to a Chebyshev polynomial
        @param[in] mat Matrix operator
        @param[in] out Output spinor
        @param[in] in Input spinor
     */
-    void matVec(const DiracMatrix &mat, ColorSpinorField &out, const ColorSpinorField &in);
+    //void chebyOp(const DiracMatrix &mat, ColorSpinorField &out, const ColorSpinorField &in);
```

Comment:
can this line be deleted?

Reply:
6df70599df5a0faaec4a5e894c113898698f2dda